tasks/fix-cat2.yml

```diff
@@ -811,48 +811,55 @@
       - home
 
 - name: "MEDIUM | RHEL-07-020620 | PATCH | All local interactive user home directories defined in the /etc/passwd file must exist."
-  command: "true"
-  changed_when: no
+  file:
+    path: "{{ item.split(':')[2] }}"
+    state: directory
+  with_items: "{{ rhel7stig_passwd_homedir_check.stdout_lines }}"
   when: rhel_07_020620
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-020620
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-020630 | PATCH | All local interactive user home directories must have mode 0750 or less permissive."
-  command: "true"
-  changed_when: no
+  file:
+    path: "{{ item.split(':')[2] }}"
+    mode: "{{ rhel7stig_homedir_mode }}"
+    state: directory
+  with_items: "{{ rhel7stig_passwd_homedir_check.stdout_lines }}"
   when: rhel_07_020630
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-020630
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-020640 | PATCH | All local interactive user home directories must be owned by their respective users."
-  command: "true"
-  changed_when: no
+  file:
+    path: "{{ item.split(':')[2] }}"
+    owner: "{{ item.split(':')[0] }}"
+    state: directory
+  with_items: "{{ rhel7stig_passwd_homedir_check.stdout_lines }}"
   when: rhel_07_020640
   tags:
       - cat2
       - medium
       - patch
-      - RHEL-07-020660
-      - notimplemented
+      - RHEL-07-020640
 
 - name: "MEDIUM | RHEL-07-020650 | PATCH | All local interactive user home directories must be group-owned by the home directory owners primary group."
-  command: "true"
-  changed_when: no
+  file:
+    path: "{{ item.split(':')[2] }}"
+    group: "{{ item.split(':')[1] }}"
+    state: directory
+  with_items: "{{ rhel7stig_passwd_homedir_check.stdout_lines }}"
   when: rhel_07_020650
   tags:
       - cat2
       - medium
       - patch
-      - RHEL-07-020670
-      - notimplemented
+      - RHEL-07-020650
```

Comment:
Good catch. strange how the tags are mis-named...

Reply:
I thought I saw an earlier note that tags would be deprecated in the long run, but no reason to have them wrong in the interim.

Reply:
I must have missed that note.  I find the per-task tags very helpful, especially during development of this role...  the 'cat2', 'medium', 'patch', 'audit', etc. are less useful to me.